cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
I'm happy to rename it to _properties_region, but I would like to keep the functionality in the Base class so that any provider can use it